docs/stencil-docs/reference-docs/styles-and-properties.md

```diff
@@ -1,5 +1,7 @@
 # Theme Styles and Properties
- 
+Use the following data types in the schema.json file to create configurable options in the [Theme Styles](https://support.bigcommerce.com/s/article/Page-Builder?language=en_US#styles) area of [Page Builder](https://support.bigcommerce.com/s/article/Page-Builder?language=en_US). Each widget template schema uses the i18n.{SettingName} internationalization format within labels to map settings to files. i18n's interpolation functionality makes it possible to integrate dynamic values into a theme. 
```

Comment:
```suggestion
Use the following data types in the `schema.json` file to create configurable options in the [Theme Styles](https://support.bigcommerce.com/s/article/Page-Builder?language=en_US#styles) area of [Page Builder](https://support.bigcommerce.com/s/article/Page-Builder?language=en_US). Each widget template schema uses the `i18n.{SettingName}` internationalization format within labels to map settings to files. i18n's interpolation functionality makes it possible to integrate dynamic values into a theme. 
```

- The json filename definitely needs code formatting, not positive about `i18n.{SettingName}`. 
- You could probably leave out the inline link to "page builder" since the one before (theme styles) takes you to the same page.